src/DetailsView/components/export-dialog.tsx

```diff
@@ -38,7 +41,8 @@ export const ExportDialog = NamedSFC<ExportDialogProps>('ExportDialog', props =>
         props.onDescriptionChange(value);
     };
 
-    const encodedHtml = encodeURIComponent(props.html);
+    const blob = props.deps.provideBlob([props.html], 'text/html');
```

Comment:
As per our conversation:
- `ExportDialog` should not know we're creating a Blob because that's an implementation detail
- We could move this 2 lines (creating the blob, creating a url for the blob) to a report provider (or file provider, the name can be generic enough that conveys we can create a url for any kind of content).

By doing this, we allow the implementation to change (maybe it's not a blob anymore, it's something else) without affecting the `ExportDialog` component.

I'm not too worried about this right now, so up to you if you want to make this change as part of this PR.

Reply:
We're going to submit for now to get it into Canary, but will put in additional changes in a second PR! Thank you!